bokeh/resources.py

```diff
@@ -66,12 +69,16 @@ def _get_cdn_urls(version=None, minified=True):
 
 def _get_server_urls(root_url, minified=True):
     _min = ".min" if minified else ""
-    result = {
-        'js_files'  : ['%sbokehjs/static/js/bokeh%s.js' % (root_url, _min)],
-        'css_files' : ['%sbokehjs/static/css/bokeh%s.css' % (root_url, _min)],
+
+    def mk_url(comp, tp):
+        _comp = '-' + comp if comp else ''
```

Comment:
Yes, that's a bug. Originally `comp` was one of `''` or `'widgets'`. I moved the `bokeh-` prefix, make this more readable. The fun part is that server examples work. Why? Because, server doesn't use resources. I'm not sure really what uses this.